hack/crds-sync.sh

```diff
@@ -17,7 +17,7 @@ for crd in "${CRDS_MAPPING[@]}" ; do
 done
 
 #this one goes in manifests rather than install, but should it? 
+cp "vendor/github.com/openshift/api/config/v1alpha1/0000_10_config-operator_01_clusterimagepolicy-TechPreviewNoUpgrade.crd.yaml" "install/0000_10_config-operator_01_clusterimagepolicy-TechPreviewNoUpgrade.crd.yaml"
```

Comment:
technically this might conflict with the MCO api PR (and the locations will be moving) but for now this should be ok